content/applications/websites/website/publish.rst

```diff
@@ -7,7 +7,7 @@ Publish
 .. toctree::
    :titlesonly:
 
-   publish/domain_name
+   /administration/maintain/domain_names
```

Comment:
This is a very bad idea.

- Having a page referenced in two different toctrees implies all sorts of technical issues that must be addressed one by one when discovered. Although we already addressed some when revamping the docs, it would be better not to try to discover new ones.
- This is ugly.
- This is very weird to navigate to Install and Maintain > Maintain > Domain names and land in User Docs > Websites > Website > Publish > Domain names instead. Sphinx is not capable of prioritizing a toctree over another.

Please find another solution for this. For example, add a short intro in "Publish" and mention the page "Domain names". Or try and play with shared files (`.. include::` directive) to see it if could suit your needs.

Reply:
I agree, but I hoped that @Lau-Odoo would fix this in the future when he'll revamp the website doc :D

Reply:
> I agree, but I hoped that @Lau-Odoo would fix this in the future when he'll revamp the website doc :D

That's, indeed, something we agreed on.

Reply:
Technical safety guideline: don't introduce crap (no offense) and *hope* someone will remember to fix it soon :D 

If we can live with the proposed structure for a short (:crossed_fingers:) while, we can also live with any other suitable solution for the same period.

Reply:
I'm not introducing new crap, it's already existing crap anyway. 
This doc was already double-indexed in Website and eCommerce. Now it would merely be Maintain and Website. ^^'